pkg/template/master_template.go

```diff
@@ -355,7 +355,6 @@ systemd:
       Wants=k8s-kubelet.service
       [Service]
       Type=oneshot
-      RemainAfterExit=yes
```

Comment:
this has to be removed so timer see it as (exited/inactive)  otherwise timer would not do anything